aim/web/ui/src/config/table/tableConfigs.ts

```diff
@@ -19,4 +19,7 @@ export const rowCeilSizeConfig = {
   },
 };
 
+export const tableColoredSelectionColumnWidth = 50;
```

Comment:
@VkoHov what do you think about changing name to uppercase and separate with underlines. 
For instance COLORED_SELECTION_COLUMN_WIDTH.